src/popper/commands/cmd_translate.py

```diff
@@ -0,0 +1,48 @@
+import click
+import os
+
+from popper.cli import pass_context, log
+from popper.parser import WorkflowParser
+from popper.translators.translater import WorkflowTranslator
+
+
+@click.command(
+    "translate",
+    short_help="Translate workflow configuration files of different runners.",
+)
+@click.option(
+    "infmt",
+    "--from",
+    help="Format of the input file.",
+    type=click.Choice(["popper"]),
+    default="popper",
+)
+@click.option(
+    "infile",
+    "--file",
+    "-f",
+    help="File containing the definition of the workflow.",
+    required=False,
+    default=".popper.yml",
+)
+@click.option(
+    "outfmt",
+    "--to",
+    help="Format of the output file.",
+    type=click.Choice(["drone"]),
+    default="drone",
+)
+@click.option(
+    "outfile", "--out", help="Name of the output file.", required=False,
+)
+@pass_context
```

Comment:
Remove this `--out` flag for simplicity and impose filename to `drone.yml` for drone and `taskfile.yml` for taskfile

Reply:
resolved 6c5917fdb882bbf684faa871a38a3fea6d06a9fb